arch/m68k/atari/config.c

```diff
@@ -374,6 +374,7 @@ void __init config_atari(void)
 	}
 	pr_cont("\n");
 
+#ifndef	CONFIG_M68000
```

Comment:
In principle, those compile time checks could be turned into runtime checks. However, since some of the runtime checks contain code that does not compile on the 68000, this was not possible. So a solution or a comment for this problem needs to be found (TODO).